pkg/cliplugins/claims/plugin/edit.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kcp-dev/logicalcluster/v2"
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	apiv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	kcpclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
+	"github.com/kcp-dev/kcp/pkg/cliplugins/base"
+	pluginhelpers "github.com/kcp-dev/kcp/pkg/cliplugins/helpers"
+)
+
+// EditAPIBindingOptions contains the options for fetching claims
+// and their status corresponding to a specific API Binding.
+type EditAPIBindingOptions struct {
+	*base.Options
+
+	// Name of the APIbinding whose claims we need to list.
+	APIBindingName string
+
+	// If allBindings is true, then get permission claims for all
+	// APIBindings in a workspace.
+	allBindings bool
+}
+
+func NewEditAPIBindingOptions(streams genericclioptions.IOStreams) *EditAPIBindingOptions {
+	return &EditAPIBindingOptions{
+		Options: base.NewOptions(streams),
+	}
+}
+
+func (e *EditAPIBindingOptions) Complete(args []string) error {
+	if err := e.Options.Complete(); err != nil {
+		return err
+	}
+
+	if len(args) > 0 {
+		e.APIBindingName = args[0]
+	}
+	return nil
+}
+
+func (e *EditAPIBindingOptions) Validate() error {
+	if e.APIBindingName == "" {
+		e.allBindings = true
+	}
+	return e.Options.Validate()
+}
+
+func (e *EditAPIBindingOptions) BindFlags(cmd *cobra.Command) {
+	e.Options.BindFlags(cmd)
+}
+
+func (e *EditAPIBindingOptions) Run(ctx context.Context) error {
+	cfg, err := e.ClientConfig.ClientConfig()
+	if err != nil {
+		return err
+	}
+
+	_, currentClusterName, err := pluginhelpers.ParseClusterURL(cfg.Host)
```

Comment:
It looks like we're copy-pasting this parse + kubeconfig -> client code in every plugin. Why? Can we make a library and re-use it?